src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,1591 @@
+
+#include <HybridEvapCoolingModel.hh>
+
+#include <UtilityRoutines.hh>
+
+#include <string>
+#include <list>
+#include <math.h>
+#include <windows.h>
+#include <ScheduleManager.hh>
+#include <General.hh>
+#include <CurveManager.hh>
+#include <DataGlobals.hh>
+#include <DataGlobalConstants.hh>
+#include <DataHVACGlobals.hh>
+#include <Psychrometrics.hh>
+#include <DataEnvironment.hh>
+#include <DataZoneEquipment.hh>
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+#include <ObjexxFCL/Fmath.hh>
+using namespace std;
+
+namespace EnergyPlus {//***************
+
+	namespace HybridEvapCoolingModel {
+		using namespace std;
+		using DataHVACGlobals::SysTimeElapsed;
+		using DataHVACGlobals::NumOfSysTimeSteps;
+		using DataHVACGlobals::NumOfSysTimeStepsLastZoneTimeStep;
+		using DataHVACGlobals::LimitNumSysSteps;
+		using DataHVACGlobals::TimeStepSys;
+		using DataGlobals::SecInHour;
+		using namespace EnergyPlus::DataEnvironment;
+		using namespace Psychrometrics;
+		using CurveManager::GetCurveIndex;
+		using CurveManager::GetCurveType;
+		using CurveManager::GetCurveMinMaxValues;
+		using CurveManager::CurveValue;
+		using EnergyPlus::CurveManager::GetNormalPoint;
+		using ScheduleManager::GetCurrentScheduleValue;
+		typedef int* MyType;
+		map<string, string> testop;
+#define  DEF_Tdb 0
+#define  DEF_RH 1
+
+#define  TEMP_CURVE 0 
+#define  W_CURVE 1
+#define  POWER_CURVE 2
+		// constructor of Model,
+
+#include <fstream>
+		fstream logfile;
+		//ZoneHybridUnitaryACSystem
+
+		bool CMode::InitializeOutsideAirTemperatureConstraints(double min, double max)
```

Comment:
1. There was a good reason for that. The values for those variables were originally collected in HybridUniaryAirConditioners.cpp and so would not have access to the private member variables, but then I moved the parsing of all the different modes to a member function within CMode, making the function calls mostly unnecessary. However they do provide a nice location to do more input validation without cluttering up the main function. As I do want to add more checks on the inputs, I think it makes sense to keep the functions.